.gitignore

```diff
@@ -1,2 +1,3 @@
 .gradle
-build
\ No newline at end of file
+build
+/.idea/
```

Comment:
Dev environment specific ignores can be set to the whole environment: https://www.atlassian.com/git/tutorials/saving-changes/gitignore#global-git-ignore-rules

No big problem for me to keep this one line here.

Reply:
Good explanation in commit body. Commit header has a typo 'deprecte'